scalafix/rules/src/main/scala/migrate/ExplicitImplicitsRule.scala

```diff
@@ -63,21 +62,9 @@ class ExplicitImplicitsRule(g: Global) extends SemanticRule("ExplicitImplicits")
 
   def getImplicitParams(originalTree: Tree)(implicit unit: g.CompilationUnit): Option[List[String]] =
     for {
-      term       <- SyntheticHelper.getTermName(originalTree)
-      context    <- CompilerService.getContext(term, g)
-      globalTree <- getTreeFromContext(context)
-      args <- globalTree match {
-                case g.Apply(_, args) if globalTree.isInstanceOf[g.ApplyToImplicitArgs] => {
-                  val listOfArgs = args.map(_.symbol.asInstanceOf[g.Symbol])
-                  listOfArgs.map(printSymbol).sequence
-                }
-                case g.Apply(_, args) if args.nonEmpty && args.head.isInstanceOf[g.ApplyToImplicitArgs] => {
-                  val newArgs    = args.head.asInstanceOf[g.Apply].args
-                  val listOfArgs = newArgs.map(_.symbol.asInstanceOf[g.Symbol])
-                  listOfArgs.map(printSymbol).sequence
-                }
-                case _ => None
-              }
+      globalTree <- CompilerService.getContext(originalTree, g)
+      _           = println(s"globalTree = ${globalTree}")
```

Comment:
Do we need this debug? If yes then maybe we should use `scribe` rather than `println`.

Reply:
no